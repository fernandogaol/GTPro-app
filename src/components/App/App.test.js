// import React from 'react';
// import { shallow } from 'enzyme';
// import renderer from 'react-test-renderer';
// import App from './App';
// import { MemoryRouter } from 'react-router-dom';

// describe('<App />', () => {
//     it('Renders without crashing', () => {
//         shallow(<MemoryRouter><App /></MemoryRouter>);
//     });

//     it('renders correctly', () => {
//         const tree = renderer
//             .create(<MemoryRouter><App /></MemoryRouter>)
//             .toJSON();
//         expect(tree).toMatchSnapshot();
//     });
// });