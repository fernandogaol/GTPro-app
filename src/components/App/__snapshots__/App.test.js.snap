// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<App /> renders correctly 1`] = `
<div
  className="App"
>
  <header
    className="App__header"
  >
    <div
      id="logo"
    />
    <div>
      <button
        id="roundButton"
        onMouseDown={[Function]}
      >
         
      </button>
      <div
        className="hide"
        id="flyoutMenu"
        onMouseDown={[Function]}
      >
        <header
          id="nav-header"
        >
          <a
            href="/"
            onClick={[Function]}
          >
            GT Pro
          </a>
        </header>
        <div
          className="Header__not-logged-in"
        >
          <a
            href="/"
            onClick={[Function]}
          >
            <svg
              aria-hidden="true"
              className="svg-inline--fa fa-home fa-w-18 navIcon home-icon"
              data-icon="home"
              data-prefix="fas"
              focusable="false"
              role="img"
              style={Object {}}
              viewBox="0 0 576 512"
              xmlns="http://www.w3.org/2000/svg"
            >
              <path
                d="M280.37 148.26L96 300.11V464a16 16 0 0 0 16 16l112.06-.29a16 16 0 0 0 15.92-16V368a16 16 0 0 1 16-16h64a16 16 0 0 1 16 16v95.64a16 16 0 0 0 16 16.05L464 480a16 16 0 0 0 16-16V300L295.67 148.26a12.19 12.19 0 0 0-15.3 0zM571.6 251.47L488 182.56V44.05a12 12 0 0 0-12-12h-56a12 12 0 0 0-12 12v72.61L318.47 43a48 48 0 0 0-61 0L4.34 251.47a12 12 0 0 0-1.6 16.9l25.5 31A12 12 0 0 0 45.15 301l235.22-193.74a12.19 12.19 0 0 1 15.3 0L530.9 301a12 12 0 0 0 16.9-1.6l25.5-31a12 12 0 0 0-1.7-16.93z"
                fill="currentColor"
                style={Object {}}
              />
            </svg>
            Home
          </a>
          <a
            href="/register"
            onClick={[Function]}
          >
             
            <svg
              aria-hidden="true"
              className="svg-inline--fa fa-user-alt fa-w-16 navIcon register-icon"
              data-icon="user-alt"
              data-prefix="fas"
              focusable="false"
              role="img"
              style={Object {}}
              viewBox="0 0 512 512"
              xmlns="http://www.w3.org/2000/svg"
            >
              <path
                d="M256 288c79.5 0 144-64.5 144-144S335.5 0 256 0 112 64.5 112 144s64.5 144 144 144zm128 32h-55.1c-22.2 10.2-46.9 16-72.9 16s-50.6-5.8-72.9-16H128C57.3 320 0 377.3 0 448v16c0 26.5 21.5 48 48 48h416c26.5 0 48-21.5 48-48v-16c0-70.7-57.3-128-128-128z"
                fill="currentColor"
                style={Object {}}
              />
            </svg>
            Register
          </a>
          <a
            href="/login"
            onClick={[Function]}
          >
            <svg
              aria-hidden="true"
              className="svg-inline--fa fa-sign-in-alt fa-w-16 navIcon login-icon"
              data-icon="sign-in-alt"
              data-prefix="fas"
              focusable="false"
              role="img"
              style={Object {}}
              viewBox="0 0 512 512"
              xmlns="http://www.w3.org/2000/svg"
            >
              <path
                d="M416 448h-84c-6.6 0-12-5.4-12-12v-40c0-6.6 5.4-12 12-12h84c17.7 0 32-14.3 32-32V160c0-17.7-14.3-32-32-32h-84c-6.6 0-12-5.4-12-12V76c0-6.6 5.4-12 12-12h84c53 0 96 43 96 96v192c0 53-43 96-96 96zm-47-201L201 79c-15-15-41-4.5-41 17v96H24c-13.3 0-24 10.7-24 24v96c0 13.3 10.7 24 24 24h136v96c0 21.5 26 32 41 17l168-168c9.3-9.4 9.3-24.6 0-34z"
                fill="currentColor"
                style={Object {}}
              />
            </svg>
            Log in
          </a>
        </div>
      </div>
    </div>
  </header>
  <main
    className="App__main"
  >
    <div
      className="landing__page"
    >
      <div
        className="logo"
      />
      <section
        className="section-app section-1"
      >
        <h3>
          <span
            className="gtpro-welcome"
          >
            Welcome to GT Pro
          </span>
          , your most efficient and practical project management application. Keep track of your to do list, vacation plans, and more. Start managing your projects now!
           
        </h3>
        <a
          href="/register"
          onClick={[Function]}
        >
          Register now for free!
        </a>
        <div
          className="managing-projects"
        />
      </section>
      <div
        className="learn-more"
      >
        <h2
          className="learn-tricks"
        >
          Learn all the tricks before you get started. GTPro makes it easy to manage
        </h2>
        <ul>
          <li>
            Work tasks and duties
          </li>
          <li>
            Team projects
          </li>
          <li>
            Homework
          </li>
          <li>
            School projects
          </li>
          <li>
            Kanban systems
          </li>
          <li>
            and more
          </li>
        </ul>
        <span
          className="slide-tolearn"
        >
           
          Learn how it works by sliding through the slides below. Try it out with our demo user. 
          <b />
          Username: demouser
          <br />
          Password: Demouser123
        </span>
      </div>
      <section
        className="dashboard section-2"
      >
        <p>
          As soon as you login, start being creative
        </p>
      </section>
      <section
        className="first-project section-3"
      >
        <p>
          Create your first project in seconds
        </p>
      </section>
      <section
        className="create-lists section-4"
      >
        <p>
          Create lists to start keeping track of tasks
        </p>
      </section>
      <section
        className="create-cards section-5"
      >
        <p>
          Add cards to keep track of ideas and progress
        </p>
      </section>
      <section
        className="sell-out"
      >
        <h3
          className="sell-out-h3"
        >
          Make sure to sign up and invite your friends and family members. We are counting on you!
          <br />
          <br />
          GTPro is a platform that offers productivity with simplicity. Team up, or use it alone. GTPro - be producitve and creative your way!
        </h3>
      </section>
    </div>
  </main>
  <footer />
</div>
`;
